src/origyn_nft_reference/migrations/lib.mo

```diff
@@ -17,6 +17,7 @@ module {
     return switch (state) {
```

Comment:
line 8 + 13: forgot to add upgrade/downgrade methods... you should "dfx deploy" locally to avoid such cases

Reply:
I think I caught this in another build when I finally tested :). [src/origyn_nft_reference/migrations/lib.mo
](https://github.com/ORIGYN-SA/origyn_nft/pull/32/commits/97cbf994c738dafae96d899835b480008fc22b9c)